compiler/src/iree/compiler/Codegen/Common/OptimizeVectorTransferPass.cpp

```diff
@@ -134,13 +135,23 @@ struct OptimizeVectorTransferPass
     // forwarding.
     eraseDeadAllocAndStores(funcOp);
   }
+  LogicalResult initializeOptions(StringRef options) override {
+    if (failed(Pass::initializeOptions(options))) {
+      return failure();
+    }
+    flatten |= optionFlatten;
```

Comment:
It would be useful to add some comments explaining why such behavior so it's easier for others to understand.

Reply:
Done.